pkg/relayer/config/types.go

```diff
@@ -66,13 +70,14 @@ type YAMLRelayMinerMetricsConfig struct {
 // YAMLRelayMinerSupplierConfig is the structure used to unmarshal the supplier
 // section of the RelayMiner config file
 type YAMLRelayMinerSupplierConfig struct {
-	ListenUrl             string                              `yaml:"listen_url"`
-	ServiceConfig         YAMLRelayMinerSupplierServiceConfig `yaml:"service_config"`
-	ServiceId             string                              `yaml:"service_id"`
-	SigningKeyNames       []string                            `yaml:"signing_key_names"`
-	RequestTimeoutSeconds uint64                              `yaml:"request_timeout_seconds"`
-	MaxBodySize           string                              `yaml:"max_body_size"`
-	XForwardedHostLookup  bool                                `yaml:"x_forwarded_host_lookup"`
+	ListenUrl             string                                         `yaml:"listen_url"`
+	ServiceConfig         YAMLRelayMinerSupplierServiceConfig            `yaml:"service_config"`
+	RPCTypeServiceConfigs map[string]YAMLRelayMinerSupplierServiceConfig `yaml:"rpc_type_service_configs"`
```

Comment:
Please update the docs with the new configuration: https://dev.poktroll.com/operate/configs/relayminer_config?_highlight=relayminer